### PR TITLE
add gcloud command

### PR DESCRIPTION
**Pull request by juanmrad**
Title says it.

Adding gcloud command into submodules so all projects can use the dockerized gcloud command on all projects.

### CODE REVIEW

File: bin/get_images [hunk 1]
```diff
@@ -17,6 +17,11 @@ export GHERKIN_CS_FIXER_REPO=gherkin-cs-fixer
 export GHERKIN_CS_FIXER_TAG=04-23-2020
 export GHERKIN_CS_FIXER_IMAGE=$DOCKER_REGISTRY_HOST/$DOCKER_REGISTRY_ACCOUNT/$GHERKIN_CS_FIXER_REPO:$GHERKIN_CS_FIXER_TAG
 
+export GCLOUD_SDK_ACCOUNT=chekote
+export GCLOUD_SDK_REPO=google-cloud-sdk
+export GCLOUD_SDK_TAG=330.0.0
```
Comment (Chekote) [inline bin/get_images:22]: I just updated the image, so we should use the latest version here.

```suggestion
export GCLOUD_SDK_TAG=334.0.0
```

File: bin/configure_gcloud.sh [hunk 2]
```diff
@@ -4,38 +4,29 @@ set -euo pipefail
...
-echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] http://packages.cloud.google.com/apt cloud-sdk main" | sudo tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
-
-echo "Importing the Google Cloud Platform public key..."
-curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key --keyring /usr/share/keyrings/cloud.google.gpg add -
-
-echo "Updating the package list and installing the Cloud SDK..."
-sudo apt-get update && sudo apt-get install google-cloud-sdk=$GCLOUD_SDK_VERSION
-
 echo "Change owner and group for gcloud config dir..."
 sudo chown -R circleci:circleci $HOME/.config/gcloud
```
Comment (jose-ochoa) [inline bin/configure_gcloud.sh:8]: Missed 
image

File: bin/gcloud [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+docker run \
+    ${LOCAL_USER_ID:-} \
+    -i$TTY \
+    --rm \
+    --mount source=$PROJECT_VOLUME,target=/workdir \
+    --mount source=user_config,target=/home/user/.config \
+    --mount source=user_docker,target=/home/user/.docker \
+    -w /workdir \
+    $GCLOUD_SDK_IMAGE gcloud "$@"
```
Comment (Chekote) [inline bin/gcloud:21]: After I implemented both gcloud and gsutil, I realized we could be more DRY by having three files:

`bin\google-cloud-sdk`:
```shell
#!/usr/bin/env bash

set -euo pipefail

dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../" && pwd )"

. ${dir}/get_tty
. ${dir}/get_local_user
. ${dir}/get_images
. "$ROOT"/docker/lib/volumes.sh

docker run \
    ${LOCAL_USER_ID:-} \
    -i$TTY \
    --rm \
    --mount source=$PROJECT_VOLUME,target=/workdir \
    --mount source=user_config,target=/home/user/.config \
    --mount source=user_docker,target=/home/user/.docker \
    -w /workdir \
    $GCLOUD_SDK_IMAGE "$@"
```

`bin\gcloud`:
```shell
#!/usr/bin/env bash

dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"

${dir}/google-cloud-sdk gcloud "$@"
```

`bin\gsutil`:
```shell
#!/usr/bin/env bash

dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"

${dir}/google-cloud-sdk gsutil "$@"
```

**Review by Chekote — APPROVED** [review]

**Review by Chekote — CHANGES_REQUESTED** [review]
Sorry. meant to request chnges.

File: bin/configure_gcloud.sh [hunk 4]
```diff
@@ -4,38 +4,29 @@ set -euo pipefail
...
 echo "Activate configurations..."
-gcloud config configurations create $GCLOUD_CONFIGURATION_NAME --activate
+gcloud config configurations create "$GCLOUD_CONFIGURATION_NAME" --activate
 
 echo "Configure account email..."
-gcloud config set account $GCLOUD_SERVICE_EMAIL
+gcloud config set account "$GCLOUD_SERVICE_EMAIL"
 
 echo "configure compute zone..."
 gcloud config set compute/zone us-central1-b
```
Comment (jose-ochoa) [inline bin/configure_gcloud.sh:23]: `us-central1-b` should be passed in since not all projects may be in this compute zone

**Review by jose-ochoa — CHANGES_REQUESTED** [review]

**Review by fortoulrodolfo — CHANGES_REQUESTED** [review]
Looking at it after Donald suggestions is implemented

**Review by jose-ochoa — APPROVED** [review]

**Review by fortoulrodolfo — APPROVED** [review]

**Review by Chekote — APPROVED** [review]

**Pull request merged**